Extensions/DraggableBehavior/tests/draggableruntimebehavior.spec.js

```diff
@@ -96,6 +104,70 @@ describe('gdjs.DraggableRuntimeBehavior', function () {
       expect(object.getY()).to.be(700);
     });
 
+    it('can drag an object without collision mask check', function () {
+      object.setPosition(450, 500);
+      object.rotateTowardAngle(45, 0, runtimeScene);
```

Comment:
Any reason you're not using `setAngle` instead of `rotateTowardAngle`?

Reply:
Wow, I somehow did not see that the method existed 😅